src/main/cpp/t03_both.cpp

```diff
@@ -23,9 +23,23 @@
 
 #include "t03_both.h"
 #include <iostream>
+#include <set>
 
 using namespace std;
 
 int t03_both() {
-
+    set <int> s;
+    int n,x,counter;
+    cin >> n;
+    for (int i =0; i < n; i++) {
+        cin >> x;
+        s.insert(x);
+    };
+    cin >> n;
+    for (int i = 0; i < n; i++) {
+        cin >> x;
+        if (s.find(x)) counter++;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Suspicious condition. The result of find() is an iterator, but it is not properly checked.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4763739)